src/main/java/com/xpandit/plugins/xrayjenkins/Utils/ProxyUtil.java

```diff
@@ -12,14 +12,20 @@
 import org.apache.http.impl.client.BasicCredentialsProvider;
 
 import javax.annotation.Nullable;
+
+import java.util.Collections;
+import java.util.List;
 import java.util.Optional;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class ProxyUtil {
```

Comment:
Just to let you know that I will create later a new unit test class to cover these changes.

Reply:
> Just to let you know that I will create later a new unit test class to cover these changes.

Yeah, I must admit I've been a bit lazy :)